keyboards/chromatonemini/keymaps/party/rules.mk

```diff
@@ -0,0 +1,4 @@
+RGB_MATRIX_ENABLE = yes       # Use RGB matrix (Don't enable this when RGBLIGHT_ENABLE is used.)
+RGB_MATRIX_DRIVER = WS2812    #
```

Comment:
Probably should put this line in the keyboard level rules.mk, so that keymaps only have to specify `RGB_MATRIX_ENABLE = yes` to turn RGB Matrix on.

Reply:
I keep the 'default' keymap without LEDs. That is why others have to be stated in each keymap level.

Reply:
Does the board only support WS2812 LEDs for RGB Matrix? If so, then you can place the `RGB_MATRIX_DRIVER` line at the keyboard level, and users that want to use RGB Matrix just have to put `RGB_MATRIX_ENABLE = yes` in their rules.mk - they don't need to know what the LED driver is.

Reply:
Oh, I see. That makes sense.
I moved the row `RGB_MATRIX_DRIVER = WS2812    #` to keyboard level rules.mk.